developer/src/kmldmlc/test/fixtures/basic.txt

```diff
@@ -58,20 +58,61 @@ block(sect)                         #  struct COMP_KMXPLUS_SECT {
   #    KMX_DWORD sect;           // 0010+ Section identity
   #    KMX_DWORD offset;         // 0014+ Section offset relative to dpKMXPlus of section
 
-  73 74 72 73
-  diff(sect,strs)
+  6b 65 79 73
+  diff(sect,keys)
+
+  6c 6f 63 61
+  diff(sect,loca)
 
   6d 65 74 61
   diff(sect,meta)
 
   6e 61 6d 65
   diff(sect,name)
 
-  6c 6f 63 61
-  diff(sect,loca)
+  73 74 72 73
+  diff(sect,strs)
 
-  6b 65 79 73
-  diff(sect,keys)
+block(keys)                         # struct COMP_KMXPLUS_KEYS {
+  6b 65 79 73                       #   KMX_DWORD header.ident;   // 0000 Section name - keys
+  sizeof(keys)                      #   KMX_DWORD header.size;    // 0004 Section length
+  02 00 00 00                       #   KMX_DWORD count;          // 0008 number of keys
+  00 00 00 00                       #   KMX_DWORD reserved;       // 000C padding
+                                    # };
+
+# Keys data:
+
+  c0 00 00 00  00 00 00 00  index(strNull,strKey1,2)  01 00 00 00   # KMX_DWORD vkey, mod, to, flags;
+  31 00 00 00  00 00 00 00  index(strNull,strKey2,2)  01 00 00 00   # KMX_DWORD vkey, mod, to, flags;
+
+block(loca)                         # struct COMP_KMXPLUS_LOCA {
+  6c 6f 63 61                       #   KMX_DWORD header.ident;   // 0000 Section name - loca
+  sizeof(loca)                      #   KMX_DWORD header.size;    // 0004 Section length
+  01 00 00 00                       #   KMX_DWORD count;          // 0008 number of locales
+  00 00 00 00                       #   KMX_DWORD reserved;       // 000C padding
+  index(strNull,strLocale,2)        #   KMXPLUS_STR locale;       // 0010+ locale string entry = 'mt'
+                                    # };
+
+block(meta)                         # struct COMP_KMXPLUS_META {
+  6d 65 74 61                       #   KMX_DWORD header.ident;   // 0000 Section name - meta
+  sizeof(meta)                      #   KMX_DWORD header.size;    // 0004 Section length
+  index(strNull,strName,2)          #   KMXPLUS_STR name;
```

Comment:
is this still used with the new 'name` section?

Reply:
I had a mental to-do item to ask about removing from meta.